### DISCUSSION TITLE
Matcher and Header

### DISCUSSION THREAD

**Discussion by fbruno93** [pactumjs/pactum#218] [Q&A]
Hi ! 

I need to match some numric header like `age: > 10`
So i tried : 

```feature
And I expect header "age" > 3
```

```js
Then(`I expect header {string} > {int}`, function (name, value) {
    this.spec.response().to.have.header(name, gte(value) )
})
```

But i've this error 
```bash
TypeError: expectedHeaderValue.toLowerCase is not a function
```

Is there a way to use match for headers values ? 

Thx :)

**Comment by ASaiAnudeep** [comment]
Hello @fbruno93,

Currently pactum doesn't support matchers while asserting on headers. I do think this is a good use case and will be creating feature request based on this thread.

Moreover, the headers are often read as key-value pairs of strings. Need to check how to auto parse them.
